scripts/validate_docstrings.py

```diff
@@ -422,10 +422,11 @@ def needs_summary(self):
 
     @property
     def doc_parameters(self):
-        return collections.OrderedDict(
-            (name, (type_, "".join(desc)))
-            for name, type_, desc in self.doc["Parameters"]
-        )
+        parameters = collections.OrderedDict()
+        for names, type_, desc in self.doc["Parameters"]:
+            for name in names.split(", "):
```

Comment:
Can you just do a `re.findall` here instead? I don't think whitespace is important

Reply:
I'd prefer that we're consistent with the spacing after the comma. I think it's a bit distracting having a mix of `foo, bar`, `foo,bar`, `foo ,bar`...

In a follow up could be nice to have more specific error messages when the problem is the spacing, but this looks good to me for this PR.

Reply:
OK sounds good. Would be nice to document that somewhere if it isn't already then (follow up PR is fine)

Reply:
Also not sure if that needs its own error code